docs/en/academic/custom-templates.md

```diff
@@ -1,83 +1,134 @@
 
-> For more information on reference documents, see the section on [`--reference-doc` in the Pandoc manual](https://pandoc.org/MANUAL.html#options-affecting-specific-writers).
+<!-- TO CHECK: Regarding the next line, the TOC flag and the whole Pandoc command found in the Advanced tab in the Preferences does not seem to exist anymore. Should the line be deleted? Or does this line now refer to PDF  in Preferences -> Asset Manager? If so, the --toc flag is set to false, which does not fit with the line below. -->
+
+> In general, you should keep the table of contents flag (i.e. `$toc$`) in the Pandoc command used by Zettlr (it can be edited in the Advanced tab in the Preferences). If you remove it, you will lose the ability to generate a table of contents (even if you indicate you want one).
+
+### 5. Run the Command!
+
+With all preconditions met, Zettlr will run the Pandoc command. If it runs successfully, your output file will open using your system's default application for your chosen output format (e.g. default PDF reader for a PDF document; default web browser for an HTML document). 
+
+If Pandoc exited with an error, Zettlr will show you the error message, which you can cut-and-paste into Google to troubleshoot the problem. If the error occured in LaTeX, the full LaTeX console output will be displayed. This can be quite verbose and confusing for newcomers. For example, if a LaTeX package is missing, you need to spot the relevant error message (i.e. `File <package-name>.sty is missing`) amongst all the other output text.
+
+> **Attention:**  If you use the default template and avoid using LaTeX commands in your Markdown file, you should not encounter any errors. If you do, it indicates there is a problem with the default template, which should be reported to Zettlr's maintainers. 
 
 ## Getting Started with Templating
 
-Now it's time to get to building a template! You can either use an external editor to write your template or use Zettlr itself. Zettlr allows you to write LaTeX templates directly inside it, so you don't have to use an external editor for that.
+Let's write your first LaTeX template, which can be done directly in Zettlr! When complete, your template file will be passed to Zettlr, Citeproc (if applicable), Pandoc, and then finally LaTeX. 
+
+<!-- TO CHECK: On 2.0.0.0-beta on a Linux machine, I do not see a + symbol to add a new file where I can type in the filename with extension. Also, if I select   File -> New File , A file is automatically generated with an ID as its name; it is not intuitive that one has to add some text, then try to save the file in order to be able to rename the file to, for example, 'my-template.tex'. Is this the intended way to create a new file with a specific name?
```

Comment:
Yes, by default Zettlr will immediately use ".md" as the default extension. In order to create a TeX file or a JSON file, you'd have to manually provide the extension. Might certainly be worth of improvement in the future.

Reply:
Ok. I will change instructions to match current functionality.